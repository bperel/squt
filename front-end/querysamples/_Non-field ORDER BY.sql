SELECT table1.field1
FROM table1
ORDER BY 1;