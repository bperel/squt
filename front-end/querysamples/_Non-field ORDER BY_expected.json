{
	"Warning" : {
		"Not supported" : {
			"Non-field items in ORDER BY CLAUSE" : "1"
		}
	},
	"Tables" : {
		"table1" : {
			"table1" : {
				"OUTPUT" : {
					"field1" : {
						"-1" : "field1"
					}
				}
			}
		}
	}
}